localisation/00_zuwc_united_localisation.csv

```diff
@@ -2886,6 +2886,17 @@ GetHorses_horses;horses;;;;;;;;;;;;;x
 GetEluneTheLightCap_The_Light;The Light;;;;;;;;;;;;;x
 GetPronoun_Her;Her;;;;;;;;;;;;;x
 GetPronoun_Its;Its;;;;;;;;;;;;;x
+average_lifespan;Average Lifespan;;;;;;;;;;;;;x
+average_lifespan_percent;Average Lifespan;;;;;;;;;;;;;x
+text_effect_average_lifespan_immortality;Average Lifespan is close to �GImmortality�!;;;;;;;;;;;;;x
+AUTOCEPHALOUS_PATRIARCHS;Autonomous Bishops;;;;;;;;;;;;;x
+AUTOCEPHALOUS_REL_HEAD_TITLE;Bishop;;;;;;;;;;;;;x
+AUTOCEPHALY_IRO;�YAdditional Autonomous Bishops�!;;;;;;;;;;;;;x
+REL_AUTOCEPHALY_DESC;Autonomous Bishops under Kings;;;;;;;;;;;;;x
+PENTARCHY_DRO;All $RELIGION$ kingdoms or empires also have their own autonomous Bishops. The Court Chaplain of such a kingdom or empire is also considered the Bishop of that realm. He will always do what his liege asks of him, but only the $REL_HEAD_TITLE$ can grant the Invasion Casus Belli. Each autonomous Bishop controls Excommunication within their respective realm.;;;;;;;;;;;;;x
+AUTOCEPHALOUS_REL_HEAD_TITLE_FEMALE;Bishop;;;;;;;;;;;;;x
+EVTDESC_WCDAC_9993_evil;Your trade post in [From.GetName] has been destroyed by local peacekeepers and guards during the last culling. The known cultists of the province have been executed.;;;;;;;;;;;;;x
+EVTDESC_WCDAC_9993_normal;Your trade post in [From.GetName] has been destroyed by the unholy fanatics of [From.Religion.GetHighGodName]. The local traders have been either enslaved or killed outright.;;;;;;;;;;;;;x
```

Comment:
add german: "Euer Handelsposten in [From.GetName] wurde von den unheiligen Fanatikern von [From.Religion.GetHighGodName] zerstört. Die lokalen Händler wurden entweder versklavt oder auf der Stelle getötet."